repair of merge error

dave-prosee opened the pull request:
probably deleted closing brackets when resolving merge conflict

Commit:
merge local with my forked repository

Merge branch '10.0' of git://github.com/magnuscolors/vertical-publishing into 10.0

Commit:
merge clone with personal repository

Merge branch '10.0' of https://github.com/dave-prosee/vertical-publishing into 10.0

dave-prosee commented:
@whulshof @Stephanvandenberg 
Jullie zijn wellicht bezig met herstart/problemen met bdu-accept en komen dan wellicht een fout bij het opstarten tegen (omdat sluiten haak mist). In deze PR de oplossing.
Blijkbaar heb ik bij het oplossen van een conflict de regel met de sluiten-haken verwijderd.

Stephanvandenberg commented:
Thanks Dave ik geef het door aan Willem. Ik laat de test omgeving even aan hem over - heb het idee dat er meer dan alleen dit aan de hand is.

Commit:
merge resolution

Commit:
publishing subscription order : clearer labels
- "is digital subscription?" changed into "digital only"
- "is subscription product?" changed into "is subscription"

dave-prosee committed:
Merge branch '10.0' into 10.0

dave-prosee commented:
Blocking merge opgelost: waren lege regels

dave-prosee commented:
@Stephanvandenberg 
Hi, Ik zie deze nog open staan en dacht dat je deze vandaag al had gedaan.
Tevens staat PR#24 nog open: juiste vertalingen

Stephanvandenberg commented:
No longer needed
